### DISCUSSION TITLE
Nix package

### DISCUSSION THREAD

**Discussion by MintyGitOut** [TheAssassin/AppImageLauncher#684] [Ideas]
### Pre-submit checks

- [X] I checked for similar issues beforehand, but could not find any. I could not add my proposal to any existing issue.
- [X] I am going to take the time to to fill in all the required details. I know that the bug report will be dismissed otherwise.

### Feature description

It would be nice to have a Nix package, as especially for NixOS it'd be nice since appimages suck, but this app makes them suck a lot less and currently it doesn't really work well or at all on NixOS.

### Proposed solution

 A Nix package for the app. Nix packages are not hard to create, there are over 100 thousand of them in the official repos, even for packages that are usually only in the AUR on distros like Arch.

### Alternative solutions you considered

A Flatpak or other distro-agnostic version of the app? Or maybe a Nix Flake or community package or something?

**Comment by TheAssassin** [comment]
I don't have a lot of experience with nix and barely ever used nix's "workaround" tool appimage-run. I certainly won't be providing packages upstream. But please don't hesitate to set up a downstream package.

**Comment by TheAssassin** [comment]
Converting to a discussion since upstream packaging is nothing I'd work on.
